src/main/modules/appinsights/index.ts

```diff
@@ -8,6 +8,7 @@ export class AppInsights {
     if (config.get('appInsights.instrumentationKey')) {
       appInsights.setup(config.get('appInsights.instrumentationKey'))
         .setSendLiveMetrics(true)
+        .setAutoCollectConsole(true, true)
```

Comment:
Looking at [the docs](https://docs.microsoft.com/en-us/azure/azure-monitor/app/nodejs#sdk-configuration) you might want to turn on `.setAutoCollectExceptions(true)` as well.